src/corehost/build.proj

```diff
@@ -55,10 +55,19 @@
 
     <!-- Generate Version files -->
     <ItemGroup>
-      <HostFiles Include="dotnet;hostfxr;hostpolicy" />
+      <HostFiles Include="dotnet">
+       <AssemblyName>.NET Core Host</AssemblyName>
```

Comment:
Is assembly name the correct name for this metadata? It looks more like a description. The name might cause some confusion.